src/main/java/frc/lib/math/Conversions.java

```diff
@@ -0,0 +1,126 @@
+package frc.lib.math;
+
+import com.revrobotics.RelativeEncoder;
+
+public class Conversions {
+
+    /**
+     * @param positionCounts CANCoder Position Counts
+     * @param gearRatio Gear Ratio between CANCoder and Mechanism
+     * @return Degrees of Rotation of Mechanism
+     */
+    public static double CANcoderToDegrees(double positionCounts, double gearRatio) {
+        return positionCounts * (360.0 / (gearRatio * 4096.0));
```

Comment:
It wouldn't hurt to document what some of these magic numbers are for.  I'm assuming they are tied to hardware characteristics, but if the hardware ever changes we'd need to know what to change.